python/lsst/daf/butler/formatters/parquet.py

```diff
@@ -489,6 +571,22 @@ def numpy_to_astropy(np_array: np.ndarray) -> atable.Table:
     return Table(data=np_array, copy=False)
 
 
+def _numpy_dict_to_astropy(numpy_dict: dict[str, np.ndarray]) -> atable.Table:
```

Comment:
Are you deliberately not calling this function?

Reply:
This function should not be here, and is being removed.